manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,275 @@
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ClusterLogForwarderSpec defines the desired state of ClusterLogForwarder
+          properties:
+            outputs:
+              description: Outputs are named destinations for log messages.
+              items:
+                description: Output defines a destination for log messages.
+                properties:
+                  elasticsearch:
+                    type: object
+                  fluent:
+                    type: object
+                  insecure:
+                    description: Insecure must be true for intentionally insecure
+                      outputs. Has no function other than a marker to help avoid configuration
+                      mistakes.
+                    type: boolean
+                  name:
+                    description: Name used to refer to the output from a `pipeline`.
+                    type: string
+                  secretRef:
+                    description: "SecretRef refers to a `secret` object for secure
+                      communication. \n Client-authenticated TLS is enabled if the
+                      secret contains keys `tls.crt`, `tls.key` and `ca.crt`. Output
+                      types with password authentication will use keys `password`
+                      and `username`, not the exposed 'username@password' part of
+                      the `url`."
+                    properties:
+                      name:
+                        description: Name is unique within a namespace to reference
+                          a secret resource.
+                        type: string
+                      namespace:
+                        description: Namespace defines the space within which the
+                          secret name must be unique.
+                        type: string
+                    type: object
+                  syslog:
+                    description: 'Syslog provides optional extra properties for `type:
+                      syslog`'
+                    properties:
+                      facility:
+                        description: "Facility to set on outgoing syslog records.
+                          \n The string value may contain an integer or one of these
+                          names:     kernel user mail daemon auth syslog lpr news
```

Comment:
is it better to give a reference to RFC? rfc5424 section 6.2.1

Reply:
as per rfc and also syslog_protocol ruby plugin, the values are `audit, alert` instead of `security,console`
as per rfc the value is `clock daemon` instead of `cron`

Reply:
+1 to referencing RFC. The only truly "standard" values are the integers - the string names and descriptions vary slightly by implementation & historical version of the RFC. I guess we should follow rfc5424 for documentation, but maybe we should also  be forgiving and accept common variations, e.g. treat "audit" and "security" as equivalent.